### PR TITLE
Creat payment methods page

### LINKED ISSUE #30: the user saved cards page
**Task title**
Payment Methods

**Task description**
Put the description here (Mandatory)

**Subtasks**
Put the subtasks here if any (optional)

**Screenshots**
Capture d’écran 2023-10-15 003128

**Link to the component on Figma**
https://www.figma.com/file/4OUnD74RgIW6tsUFdb7gcW/Untitled?type=design&node-id=2-4846&mode=design&t=iYcMyvxa9t3sNrbw-4

### PR DESCRIPTION
**Pull request by Lilicod**
## Description

This pull request introduces the Payment Methods that display the user's saved cards.

## Changes Made

- Change in card colors based on the card type (Master or Visa). 
- Added a "Delete" button for each card. Users can remove cards from their saved payment methods.
- Included a 'Add new card' button, which takes the user to add new card page.

## Additional Notes

 There's an issue where card deletion causes the slider to disappear and cards to be displayed randomly. This issue is not resolved yet.

## Related Issues
fix #30 

## Screenshots
Capture d’écran 2023-11-01 163536

# Related Issue

- Resolve #30

### CODE REVIEW

File: src/components/Cards/PaymentCard.jsx [hunk 1]
```diff
@@ -17,13 +17,21 @@ export default function PaymentCard({
...
+    };
+
     const borderClasses = selected
         ? "border-8 border-Teal"
         : "border border-gray-300";
     return (
         <div
             onClick={handleCardClick}
-            className={`${borderClasses} w-[270px] h-[190px] md:w-[280px] md:h-[190px] lg:w-[380px] lg:h-[230px] bg-gradient-to-tr rounded-2xl flex content-center items-center justify-center from-[#9C2CF3] to-[#3A49F9]`}
+            className={`${borderClasses} w-[270px] h-[190px] md:w-[280px] md:h-[190px] lg:w-[380px] lg:h-[230px] bg-gradient-to-tr ${
```
Comment (dzc0d3r) [inline src/components/Cards/PaymentCard.jsx]: @Lilicod i think  w-[270px] and h-[190px] on small screens make the card too big on certain devices  .. better way is to use rem or something i guess

**Review by dzc0d3r — APPROVED** [review]
Great Job @Lilicod 
Thank you :1st_place_medal:

**Comment by dalilia** [comment]
good work @Lilicod

File: src/pages/paymentMethods/index.jsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    const { t } = useTranslation("common");
+    const router = useRouter();
+    const language = router.locale;
+
+    useEffect(() => {
+        document.body.dir = language == "ar" ? "rtl" : "ltr";
+    }, [language]);
+
+    const [cards, setCards] = useState([
+        {
```
Comment (M07am3dM) [inline src/pages/paymentMethods/index.jsx:24]: Hey @Lilicod , for this array of objects i think it's better to assigne it to variable in the util folder inside the library file and import it here to use it in the useEffect, thank you.

Comment (M07am3dM) [reply src/pages/paymentMethods/index.jsx:24]: Great work @Lilicod well done.

**Review by M07am3dM — APPROVED** [review]

**Pull request merged**